### PR TITLE
adds completed / passing tags filter to compliance entities table

### PR DESCRIPTION
**Pull request by theseyi**

### CODE REVIEW

File: wherehows-web/app/components/dataset-compliance.ts [hunk 1]
```diff
@@ -820,11 +824,12 @@ export default class DatasetCompliance extends Component {
         [TagFilter.showAll]: identity,
         [TagFilter.showReview]: tagsRequiringReview(complianceDataTypes, {
           checkSuggestions: false,
-          suggestionConfidenceThreshold
+          suggestionConfidenceThreshold: 0
         }),
         [TagFilter.showSuggested]: arrayFilter((tag: IComplianceChangeSet) =>
           tagSuggestionNeedsReview({ ...tag, suggestionConfidenceThreshold })
```
Comment (igbopie) [inline wherehows-web/app/components/dataset-compliance.ts]: Just wondering, shouldn't hey have the same value?

File: wherehows-web/app/components/dataset-compliance.ts [hunk 2]
```diff
@@ -820,11 +824,12 @@ export default class DatasetCompliance extends Component {
         [TagFilter.showAll]: identity,
```
Comment (igbopie) [inline wherehows-web/app/components/dataset-compliance.ts]: Consider changing to new computed style :)

Comment (theseyi) [reply wherehows-web/app/components/dataset-compliance.ts]: new computed style?

File: wherehows-web/app/components/dataset-compliance.ts (hunk 1 above)
Comment (theseyi) [reply wherehows-web/app/components/dataset-compliance.ts]: which values should be the same?

File: wherehows-web/app/components/dataset-compliance.ts (hunk 2 above)
Comment (theseyi) [reply wherehows-web/app/components/dataset-compliance.ts]: this is an object using the strategy pattern and computed property names to resolve to a function
```js
func = {
  [computedA]: () => A,
  [computedB]: () => B
}[valueAorB]
```
Using property getters does not really provide an enhancement for this use case and seems cumbersome to index in ts

File: wherehows-web/app/components/dataset-compliance.ts (hunk 1 above)
Comment (theseyi) [reply wherehows-web/app/components/dataset-compliance.ts]: Ah, yea, it's an parameter to filter out suggestions not meeting a threshold and has no import when the `checkSuggestions` flag is false,  but the interface does not make it optional, maybe I should

**Comment by theseyi** [comment]
@cptran777

**Pull request merged**